Export Analysis Reports/Sdl.Community.ExportAnalysisReports.UnitTests/ReportServiceUnitTests.cs

```diff
@@ -34,41 +34,44 @@ private void GetReportPath_IsSuccessfullyReturned_UnitTest()
 			Assert.NotNull(jsonPath);
 		}
 
-		[Fact]
-		private void GetReportPath_IsNotSuccessfullyReturned_UnitTest()
+		[Theory]
+		[InlineData("")]
+		private void GetReportPath_IsNotSuccessfullyReturned_UnitTest(string filePath)
 		{
 			_reportService.SaveExportPath(_exportTempFolder);
-			var jsonPath = _reportService.GetJsonReportPath(string.Empty);
+			var jsonPath = _reportService.GetJsonReportPath(filePath);
 
 			Assert.False(!string.IsNullOrEmpty(jsonPath));
```

Comment:
Just an observation:
  Here you assert that it is false that the string is not null; this is a double negation, much easier to read if you just make it positive like Assert.True(string.IsNullOrEmpty(jsonPath))

Reply:
I've updated